### ISSUE TITLE
Identify changes in ufs-v2.0.0 branch that need to be merged into develop

### ISSUE THREAD

**Issue by climbfuji** [NOAA-EMC/NCEPLIBS-external#93]
The latest public release contains updates that are not in develop. We need to go through those and make sure that whatever is needed finds its way back into develop.

The develop branches of NCEPLIBS-external and NCEPLIBS must be functional, even if they are updated only once when releases are made.

**Comment by climbfuji** [comment]
Closing. This repository is no longer maintained.
